### PR TITLE
final README updates for 200.5

### PR DESCRIPTION
**Pull request by sorenoid**
* add images to ScenView and MapView Callout apps
* update FeatureForms README for attachments and theming API
* update GeoViewCompose README for final version of Callout

### CODE REVIEW

File: microapps/MapViewCalloutApp/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # MapView Callout Micro-app
 
-This micro-app demonstrates the use of `Callout` with a composable `MapView`.
+This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
```
Comment (shubham7109) [inline microapps/MapViewCalloutApp/README.md:3]:
```suggestion
This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at a given point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
```

File: microapps/MapViewCalloutApp/README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 
 ## Usage
 
-The application starts with a `MapView`. A `Callout` is shown on the `MapView` when you tap on it.
+The application starts with a choice of two screens
+* A `MapView` with `Callout`s placed at the location of user taps on the screen.
+* A `MapView` with `Callout`s placed on GeoElements tapped by the user.
+* For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).
```
Comment (shubham7109) [inline microapps/MapViewCalloutApp/README.md:10]:
```suggestion
* For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose#display-a-callout).
```

File: microapps/SceneViewCalloutApp/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# SceneView Callout Micro-app
+
+This micro-app demonstrates the use of `Callout` with a composable `SceneView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Scene. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
+
+## Usage
+
+The application presents a screen with a `SceneView` which when tapped will show a `Callout` and a red graphic cross indicating the tapped location.
+For more information on the composable `Callout` component and how it works, see it's section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).
```
Comment (shubham7109) [inline microapps/SceneViewCalloutApp/README.md:8]:
```suggestion
For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose#display-a-callout).
```

**Review by shubham7109 — COMMENTED** [review]
The link to Display a Callout didn't work, added change note for it and minor comments on the readme wording used.

File: microapps/MapViewCalloutApp/README.md (hunk 1 above)
Comment (eri9000) [inline microapps/MapViewCalloutApp/README.md:3]:
```suggestion
This micro-app showcases the use of `Callout` with a composable `MapView `. `Callout` is a composable function that renders an empty composable Box at a specified point or GeoElement on a Map. The content of the composable Box is customizable, and the container is a stylable rectangular shape with a leader line positioned at the tap location or GeoElement passed into the `Callout` composable function.
```

File: microapps/MapViewCalloutApp/README.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
 # MapView Callout Micro-app
 
-This micro-app demonstrates the use of `Callout` with a composable `MapView`.
+This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
 
 ## Usage
 
-The application starts with a `MapView`. A `Callout` is shown on the `MapView` when you tap on it.
+The application starts with a choice of two screens
+* A `MapView` with `Callout`s placed at the location of user taps on the screen.
```
Comment (eri9000) [inline microapps/MapViewCalloutApp/README.md:8]:
```suggestion
* A `MapView` that displays a `Callout` at the location where the user taps on the screen
```

File: microapps/MapViewCalloutApp/README.md [hunk 5]
```diff
@@ -1,9 +1,13 @@
...
+* A `MapView` with `Callout`s placed on GeoElements tapped by the user.
```
Comment (eri9000) [inline microapps/MapViewCalloutApp/README.md:9]:
```suggestion
* A `MapView` that displays a `Callout` placed on the GeoElement tapped by the user.
```

File: microapps/MapViewCalloutApp/README.md (hunk 2 above)
Comment (eri9000) [inline microapps/MapViewCalloutApp/README.md:10]:
```suggestion
* For more information on the composable `Callout` component and how it works, see the section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).
```

File: microapps/SceneViewCalloutApp/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# SceneView Callout Micro-app
+
+This micro-app demonstrates the use of `Callout` with a composable `SceneView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Scene. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
```
Comment (eri9000) [inline microapps/SceneViewCalloutApp/README.md:3]:
```suggestion
This micro-app showcases the use of `Callout` with a composable `SceneView`. Callout is a composable function that renders an empty composable Box at a specified point or GeoElement on a Scene. The content of the composable Box is customizable, and the container is a stylable rectangular shape with a leader line positioned at the tap location or GeoElement passed into the `Callout` composable function.
```

File: microapps/SceneViewCalloutApp/README.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+## Usage
+
+The application presents a screen with a `SceneView` which when tapped will show a `Callout` and a red graphic cross indicating the tapped location.
```
Comment (eri9000) [inline microapps/SceneViewCalloutApp/README.md:7]:
```suggestion
The application displays a screen containing a `SceneView` that, when tapped, shows a Callout and a red graphic cross marking the tapped location.
```

**Review by eri9000 — COMMENTED** [review]
Some suggestions

File: microapps/MapViewCalloutApp/README.md (hunk 1 above)
Comment (sorenoid) [reply microapps/MapViewCalloutApp/README.md]: @eri9000 I committed @shubham7109 's suggestion and this suggestion is not out of date.

**Review by shubham7109 — APPROVED** [review]
Looks good with current changes 👍

**Comment by eri9000** [comment]
LGTM

**Pull request merged**